src/app/routes/DefaultErrorRoute.tsx

```diff
@@ -15,7 +15,7 @@ export const DefaultErrorRoute = () => {
 
     if (isRouteError) {
         title = error.statusText
-        message = error?.error?.message
+        message = error?.data?.message
```

Comment:
due to change in type definition here: https://github.com/remix-run/react-router/commit/a4495dee75166546dd92a4072fedab30abce84cf